geometry_msgs/Point point1
geometry_msgs/Point point2
bool havenopoints
